install.sh

```diff
@@ -4101,6 +4101,47 @@ setup_automatic_update_checks() {
     fi
 }
 
+############################################################
+# Find existing repositories from a start directory `$1`.
+#   Sets the variable `$EXISTING_REPOSITORY_LIST`
+#
+# Returns:
+#   0 on success, 1 on failure
+############################################################
+find_existing_git_dirs() {
+
+    REPOSITORY_LIST=$(
+        find "$1" \( -type d -and -name .git \) -or \
+            \( -type f -and -name HEAD -and -not -path "*/.git/*" \) 2>/dev/null
+    )
+
+    # List if existing Git repositories
+    EXISTING_REPOSITORY_LIST=""
+
+    IFS="$IFS_NEWLINE"
+    for EXISTING in $REPOSITORY_LIST; do
+        unset IFS
+
+        if [ -f "$EXISTING" ]; then
+            # Strip HEAD file
+            EXISTING=$(dirname "$EXISTING")
+        fi
+
+        # Go to the root git dir (works in bare and non-bare repositories)
+        GIT_DIR=$(cd "$EXISTING" && GIT_DISCOVERY_ACROSS_FILESYSTEM=0 git rev-parse --git-dir 2>/dev/null)
```

Comment:
Is the `GIT_DISCOVERY_ACROSS_FILESYSTEM=0` necessary here?

Reply:
I would put it in, because someone accidentally set that, I just disable it.